src/ci/docker/host-x86_64/dist-x86_64-linux/build-gccjit.sh

```diff
@@ -21,8 +21,9 @@ cd ../gcc-build
     --disable-bootstrap \
     --disable-multilib \
     --prefix=$(pwd)/../gcc-install
```

Comment:
Doesn't this need a backslash? Otherwise you're calling a new command `--quiet`...

Reply:
Arf, thanks!